src/account-info/log-in.js

```diff
@@ -49,31 +53,19 @@ class SignIn extends Component {
             </div>
             <div>
                 <p>Don't have an account?</p>
-                <Link to='/sign-up'>
-                    <button>Sign Up</button>
-                </Link>
+                    <button onClick={() => goToRoute(SIGN_UP)}>
+                      Sign Up
+                    </button>
             </div>
         </div>
-        
-        {isLoaded(auth) && !isEmpty(auth) && (
```

Comment:
I like this pattern but it appears in several places. I would try to figure out the minimum number of combos you need and pull them up into a constant.

Makes sense now when reading but when you come back to change something all those boolean logic combos are a good place for bugs to hide. 

```
const hasAuthUser = isLoaded(auth) && !isEmpty(auth)
```
and then 
```
hasAuthUser && <AwesomeComponent /> 
```

Whether you refactor or not it would be could to have positive/negative test cases for the different combos.